compiler/daemon/daemon-common-new/build.gradle.kts

```diff
@@ -8,8 +8,6 @@
  * that can be found in the license/LICENSE.txt file.
  */
```

Comment:
@Ololoshechkin I suppose copyrights in `build.gradle.kts` files added in your commits are also not needed